.pre-commit-config.yaml

```diff
@@ -64,10 +64,10 @@ repos:
     hooks:
       - id: isort
   - repo: https://github.com/psf/black
-    rev: 22.10.0 # Replace with any tag/version: https://github.com/psf/black/tags
+    rev: 22.12.0 # Replace with any tag/version: https://github.com/psf/black/tags
```

Comment:
Should probably not be in this PR due to being unrelated to removing Python 3.7

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/526/commits/3e370499a7edc12328fb86034c0dbe8be725b9d4